ansible/inventory/env/group_vars/all.yml

```diff
@@ -649,3 +649,4 @@ group_activity_agg_cache_enable: false
 
 # nodebb variables
 sunbird_nodebb_storage_key: "{{ core_vault_sunbird_nodebb_storage_key }}"
+mongo_nodebb_host: "{{ groups['mongo'] | join ',' }}"
```

Comment:
will the app support ',' seperated ips @vinukumar-vs

Reply:
https://community.nodebb.org/topic/10810/connection-to-mongodb-that-is-hosted-on-atlas/2